nodejs/eks/examples/tags/index.ts

```diff
@@ -67,7 +68,11 @@ const spot = new eks.NodeGroup("example-ng-tags-spot", {
             effect: "NoSchedule",
         },
     },
-    autoScalingGroupTags: { "myAutoScalingGroupTag3": "true" },
+    autoScalingGroupTags: cluster2.core.cluster.name.apply((clusterName: string) => ({
```

Comment:
Do you need the type in the lambda, or can it just be `clusterName => ({...})`?

Reply:
If I don't specify the `string` type in `.apply((clusterName: string) => ({...})`, tsserver yells at me with: `[tsserver] Parameter 'clusterName' implicitly has an 'any' type.`

Reply:
Oh, weird. I would have thought that the type would have been inferred in this case.